### ISSUE TITLE
Document open source YCSB performance with various replication factors

### ISSUE THREAD

**Issue by robert-s-lee** [cockroachdb/docs#4488]
Fixed:

regions: Ireland, London, Frankfurt
AZs: 3 AZs per region
nodes: 1 node per AZ (4 vCPUs)

Variables:
3 replication factor
5 replication factor
7 replication factor
9 replication factor

YCSB setup:
insert 10 million rows in parallel from each node.  total will be 90 million rows
the primary key prefix will be user000... user008, 
the primary key suffix will range from 0,000,000 to 9,999,999

**Comment by robert-s-lee** [comment]
3 DC, 2 AZ per DC, 5 way replication

```
export f=robert-ycsb-3d2z5r
export ver=v19.1.0-beta.20190318
roachprod create $f --geo --gce-zones \
europe-west1-b,europe-west2-a,europe-west3-a,europe-west1-c,europe-west2-c,europe-west3-b \
-n 6 --local-ssd-no-ext4-barrier 
```

<img width="668" alt="Screen Shot 2019-03-20 at 11 34 19 AM" src="https://user-images.githubusercontent.com/6315124/54697705-66e8dc80-4b04-11e9-8de1-fdc9df3995c6.png">

<img width="673" alt="Screen Shot 2019-03-20 at 11 34 02 AM" src="https://user-images.githubusercontent.com/6315124/54697719-6e0fea80-4b04-11e9-95e3-d6996d5b1e13.png">

**Comment by robert-s-lee** [comment]
4 DC, 2 AZ per DC, 3 way replication.

YCSB B: read=11.3ms update=27.5ms

```
export f=robert-ycsb-4d2z3r
export ver=v19.1.0-beta.20190318
roachprod create $f --geo --gce-zones \
europe-west1-b,europe-west2-a,europe-west3-a,europe-west4-a,europe-west1-c,europe-west2-c,europe-west3-b,europe-west4-b \
 -n 8 --local-ssd-no-ext4-barrier
```

```
100000 operations; 267.24 current ops/sec; [READ: Count=235, Max=14527, Min=1377, Avg=11316.29, 90=11935, 99=13735, 99.9=14527, 99.99=14527] [CLEANUP: Count=6, Max=330, Min=187, Avg=227.83, 90=240, 99=330, 99.9=330, 99.99=330] [UPDATE: Count=13, Max=31535, Min=10736, Avg=27544.92, 90=29087, 99=31535, 99.9=31535, 99.99=31535]
```

<img width="565" alt="Screen Shot 2019-03-20 at 3 18 46 PM" src="https://user-images.githubusercontent.com/6315124/54712849-b7236700-4b23-11e9-9c4f-0a1bd68f06e0.png">

<img width="486" alt="Screen Shot 2019-03-20 at 3 12 40 PM" src="https://user-images.githubusercontent.com/6315124/54712858-bb4f8480-4b23-11e9-8040-d29e0815b1e7.png">

<img width="1137" alt="Screen Shot 2019-03-20 at 3 04 18 PM" src="https://user-images.githubusercontent.com/6315124/54712867-bf7ba200-4b23-11e9-9f23-4f2331bdd1d9.png">

<img width="837" alt="Screen Shot 2019-03-20 at 3 22 00 PM" src="https://user-images.githubusercontent.com/6315124/54712986-f8b41200-4b23-11e9-825a-a878e09af840.png">
<img width="831" alt="Screen Shot 2019-03-20 at 3 21 49 PM" src="https://user-images.githubusercontent.com/6315124/54712987-f8b41200-4b23-11e9-807e-8075d6a5f6f1.png">

**Comment by jseldess** [comment]
@johnrk, @mwang1026, @rmloveland, closing this for now. If this topics comes out of your Product Areas' problems to solve or priorities in the future, we can create a new issue with more context.
